src/Game.java

```diff
@@ -0,0 +1,80 @@
+import java.io.InputStream;
+import java.io.PrintStream;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+/**
+ * Created by Shreyansh97 on 7/9/19
+ */
+
+public class Game {
+  private List<Player> players;
+  private Board board;
+  private Dice dice;
+  
+  private Game(int boardSize, int diceCount) {
+    board = new Board(boardSize);
+    dice = new Dice(diceCount);
+    players = new ArrayList<>();
+  }
+  
+  public static Game setupFromUser(InputStream in) {
+    Scanner scanner = new Scanner(in);
```

Comment:
why static method?

Reply:
The object is created in this method... the game is being setup using this function